sips/minutes/_posts/2016-07-15-sip-minutes.md

```diff
@@ -131,17 +135,17 @@ be resolved before resuming its evaluation.
 revisit the support of async/await in try/catch blocks. When that's considered,
 this SIP should be reopened and and it should see another round of discussion.
 
-#### Proposal SIP-20: Improved lazy val initialization by Dmitry Petrashko
+#### Proposal SIP-20: Improved lazy val initialization presented
 Sebastien reviews the SIP and asks Dmitry, present in the meeting, to correct
 him if he's wrong. He agrees that the SIP is desirable but he's unsure about the
 benchmarks and which of the proposals is faster. Dmitry explains that the
-benchmarks are in the repo, and that in the best case the initialization doesn’t
-need synchronization.
+benchmarks are in the repository.
   
-He also recommends the author to include more documentation.
+Sebastien also recommends the authors to include more documentation.
   
-**Outcome**: Dmitry volunteers to provide an implementation in three months. The
-Process Lead schedules the next iteration by October 2016.
+**Outcome**: Dmitry says he's likely to have time to update the SIP in three
```

Comment:
@heathermiller, @sjrd wrote his notes based on what @jvican wrote in this blog post.
I indeed contacted both of them privately and asked them to correct misinformation. [The source of what @srjd wrote __IS__ this post](https://gist.github.com/DarkDimius/0d5bdaa2b5500646327ac06fcea181a9).

The suggestions by Seb has been summarized in 10 seconds here: https://youtu.be/MYH0SRDLgKo?t=2465 
After this summary I'm suggesting that I will likely not be up to [`updating the sip immediately`](https://youtu.be/MYH0SRDLgKo?t=2490). And I say that I can do in in three month. It takes less that 30 seconds of video to see what did I agree on. Please, don't make false claims about what I've promised.